content/ov/delta.en.md

```diff
@@ -17,23 +17,41 @@ This is an example of gitconfig settings.
 
 ```gitconfig
 [core]
-    pager = delta
+    # delta will used as the default pager for git
+    # and ov as the default pager for delta
+    # the pager will be overloaded via the [pager] section for a few commands
+    pager = delta --pager="ov -F"
+
+[pager]
+    # overload delta pager for some commands
+    show = delta --pager="ov -F --header 3"
+
+    # we are using delta --features to use different pager per command
+    # to bring the equivalent of delta --navigate while using ov
+    # this allows to maintain a simpler config file and avoid escaping quotes
+    diff = delta --features ov-diff
+    log  = delta --features ov-log
 
 [delta]
-    navigate = true
+    navigate = false # disabled because of ov integration
```

Comment:
If this navigate is set to true, `•` will be added at the beginning of each diff, so it is included. By specifying that `•` with --pattern '•', you can move it by n/N.
If you want to disable it, `--pattern ' •' is also not necessary.

I think I need to add more explanation.

Reply:
I will give a check and come back to you

Reply:
Please let me know if you want me to open distinct PR.

One for the changes about git conf with of only

And one focused on ov integration with git+delta

Reply:
It should be ok now

I also reworked the documentation and comments

Please review